sol/Usdx.sol

```diff
@@ -1,48 +1,148 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.2;
 
-import "./chainlink/evm-contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+import "./chainlink/evm-contracts/src/v0.7/interfaces/AggregatorV3Interface.sol";
 import "./openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
 import "./openzeppelin-contracts/contracts/access/Ownable.sol";
 import "./openzeppelin-contracts/contracts/utils/math/SafeMath.sol";
 import "./openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
 
-/*   $$\   $$\  $$$$$$\  $$$$$$$\  $$\   $$\ 
+/*   $$\   $$\  $$$$$$\  $$$$$$$\  $$\   $$\
  *   $$ |  $$ |$$  __$$\ $$  __$$\ $$ |  $$ |
  *   $$ |  $$ |$$ /  \__|$$ |  $$ |\$$\ $$  |
- *   $$ |  $$ |\$$$$$$\  $$ |  $$ | \$$$$  / 
- *   $$ |  $$ | \____$$\ $$ |  $$ | $$  $$<  
- *   $$ |  $$ |$$\   $$ |$$ |  $$ |$$  /\$$\ 
+ *   $$ |  $$ |\$$$$$$\  $$ |  $$ | \$$$$  /
+ *   $$ |  $$ | \____$$\ $$ |  $$ | $$  $$<
+ *   $$ |  $$ |$$\   $$ |$$ |  $$ |$$  /\$$\
  *   \$$$$$$  |\$$$$$$  |$$$$$$$  |$$ /  $$ |
  *    \______/  \______/ \_______/ \__|  \__|
  *
  *   USDX is a USD based stablecoin with the following properties:
- *   - Eth sent to the USDX contract is minted into USDX at the current
- *     Eth/USD exchange rate.
- *   - Received eth is then burned forever.
+ *   - Eth sent to the usdx contract is locked, and USDX is minted
+ *     into the sender's account at the current eth/usd exchange rate.
+ *   - Locked eth can be redeemed by the original sender by burning USDX
+ *     at the originally minted price.
  *
- *   Note: Owner is able to set the Eth/USD oracle.
+ *   Note: Owner is able to set the eth/usd oracle.
  */
 contract USDX is ERC20, Ownable {
 	using SafeMath for uint256;
 	using SafeCast for int256;
 	uint8 constant FEED_DECS = 8;
-	// ETH/USD exchange rate feed.
-	AggregatorV3Interface public ethUsdPriceFeed;
+	AggregatorV3Interface public usdPriceFeed;
+	uint8 public feedDecs;
```

Comment:
Is this feedDecs needed? I can't find where it's used, so just wanted to check

Reply:
Good catch! Yea, this can be removed.